filters/SkewnessBalancingFilter.cpp

```diff
@@ -67,7 +67,17 @@ std::set<PointId> SkewnessBalancingFilter::processGround(PointViewPtr view)
     point_count_t n1(0);
     double delta, delta_n, term1, M1, M2, M3;
     M1 = M2 = M3 = 0.0;
-    std::vector<double> skewness(view->size());
+
+    auto setGround = [&view](PointId start, PointId end)
```

Comment:
To me it's unexpected that `end` would be inclusive here, when I see `begin`/`end` type ranges to a function I'd expect STL-like behavior which wouldn't include the `end`.  However the scope here is very localized so I could live with it.

Reply:
Is it the name you don't like?  The point is not to overflow.